CMake/External_PostgreSQL.cmake

```diff
@@ -76,7 +77,8 @@ file(APPEND ${fletch_CONFIG_INPUT} "
 ########################################
 # PostgreSQL
 ########################################
-set(PostgreSQL_ROOT @PostgreSQL_ROOT@)
-set(PostgreSQL_MAJOR_VERSION @PostgreSQL_MAJOR_VERSION@)
+set(PostgreSQL_ROOT \{PostgreSQL_ROOT})
```

Comment:
This is still incorrect. PostgreSQL_ROOT = \${fletch_ROOT} is what you're after. The Major_VERSION string was correct before.